kodilla-testing/src/main/java/com/kodilla/testing/shape/Square.java

```diff
@@ -0,0 +1,35 @@
+package com.kodilla.testing.shape;
+
+class Square implements Shape{
+
+    private double length;
+
+    public Square(double length) {
+        this.length = length;
+    }
+
+    public String getShapeName() {
+        return this.getClass().getSimpleName() + "(" + this.length + ")";
```

Comment:
GetShapeName to getShapeName a nie cos wiecej :/
Proponuje nadpisac w kazdej z klas implementujacych interfejs Shape metidy toString (w niej wykorzystac metode getShapeName)